thesis/03-implementation.tex

```diff
@@ -175,24 +174,24 @@ \subsection{Components overview}
 
 \subsubsection{Frontend}
 
-The web application contains two main subcomponents: \textit{Guide Creator} and \textit{Guide Viewer}, responsible respectively for: creating instructions and viewing animated guides. Other pages are mainly related to the community aspect and will only be covered superficially in this section.
+The web application contains two main subcomponents: \textit{Guide Creator} and \textit{Guide Viewer}, responsible respectively for creating instructions, and viewing animated guides. Other components are mainly related to the community aspect and will only be covered superficially.
 
 \begin{figure}[H]
   \caption{Overview of the frontend pages and components. Grayed out pages are related to user management.}
   \centering
     \includegraphics[width=\textwidth]{assets/3-frontend-architecture.png}
 \end{figure}
 
-The web application has been structured in a way that encourages building and reusing small components. An example would be a \textit{Viewer} component which is responsible for rendering origami figures and appears in both \textit{Guide Creator} and \textit{Guide Viewer} pages. The structure has allowed us to keep the code maintainable and implement changes quickly.
+The web application has been structured in a way that encourages building and reusing small components. An example would be a \textit{Viewer} component which is responsible for rendering origami figures and appears in both \textit{Guide Creator} and \textit{Guide Viewer} pages. This structure has allowed us to keep the code maintainable and implement changes quickly.
 
 \medskip
 
-Data storage and manipulation are powered by three stores which store data related to different aspects and functionality of the system.
+Data storage and manipulation are powered by three \textit{stores} which store data related to different aspects and functionality of the system.
 
 \begin{itemize}
-	\item The Creator store is responsible for storing an Instruction while it is being worked on.
-	\item The Player store stores information about the Guide while it is being viewed.
-	\item The Community store holds information regarding user sessions.
+	\item The \textit{Creator store} is responsible for storing an Instruction while it is being worked on.
```

Comment:
I think `the Instruction`  not  `an Instruction`. But not sure, articles are weird.
Let's at least be consistent, the next item has `the Guide`